src/hooks/core/websocket/useSimpleWebsocketHooks.tsx

```diff
@@ -64,6 +64,7 @@ const useSimpleWebsocket = (url, protocol = undefined) => {
         if (json?.message?.message_type === 'text') {
           json.message.attachments = [];
         }
+        featLog('onmessage', json);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `console.log` for logging is not recommended in production code. It's better to use a dedicated logging library that can provide different levels of logging (info, debug, error, etc.) and can be configured to disable logging for production. If you don't have a logging library, consider adding one.
```diff
-        featLog('onmessage', json);
+        logger.debug('onmessage', json);
```
Please replace `logger.debug` with your actual logging function.


<!-- This is an auto-generated comment by OSS CodeRabbit -->